python/tests/tsutil.py

```diff
@@ -1077,6 +1093,18 @@ def py_sort(tables, use_num_descendants=False):
             copy.mutations[mut_id].metadata,
             copy.mutations[mut_id].time,
         )
+    for mig_id in sorted_migs:
+        tables.migrations.add_row(
+            copy.migrations[mig_id].left,
+            copy.migrations[mig_id].right,
+            copy.migrations[mig_id].node,
+            copy.migrations[mig_id].source,
+            copy.migrations[mig_id].dest,
+            copy.migrations[mig_id].time,
+            copy.migrations[mig_id].metadata,
+        )
```

Comment:
did you mean to put the migration sort stuff in here?

Reply:
Yes, otherwise a comparison of the python and c sorting fails.